cl2-component-library/src/components/LateralScrollControls/index.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { useEffect, useState } from 'react';
+import { useTheme } from 'styled-components';
+import Box from '../Box';
+import Button from '../Button';
+import testEnv from '../../utils/testUtils/testEnv';
+interface Props {
+  containerRef?: React.RefObject<HTMLDivElement>; // Ref of the container with elements which can be laterally scrolled through
+  scrollBtnDistanceMobile?: number; // Lateral scroll distance on arrow click for mobile
+  scrollBtnDistanceDesktop?: number; // Lateral scroll distance on arrow click for desktop
```

Comment:
Let's try to come up with sensible default values for this and not expose these props, if possible.